doc/source/reference/arrays.classes.rst

```diff
@@ -75,7 +75,7 @@ NumPy provides several hooks that classes can customize:
        :func:`~numpy.matmul`, which currently is not a Ufunc, but could be
        relatively easily be rewritten as a (set of) generalized Ufuncs. The
        same may happen with functions such as :func:`~numpy.median`,
-       :func:`~numpy.min`, and :func:`~numpy.argsort`.
+       :func:`~numpy.minimum`, and :func:`~numpy.argsort`.
```

Comment:
This looks wrong, minimum _is_ a ufunc

Reply:
I was actually confused on two-three options here, NumPy doesn't have numpy.min but has numpy.minimum, numpy.amin, numpy.fmin. Not sure which one was referred here. Could you help on this?

Reply:
That seems like an error in the documentation. NumPy has `np.min` (similar to `ndarray.min()`). The issue is probably that `np.min` is an alias to `np.amin`.

I think we should probably just document `min`? (A different issue is `np.amin`, to me it feels like it is an old alias to avoid name clashes with `from numpy import *`, a pattern that is lukily used much less nowadays.)

Reply:
Honestly, this part is not really for users new to numpy, so I am fine with just making it `amin`, no matter which one we prefer to advertise more.